### ISSUE TITLE
Where are the loss weight settings?

### ISSUE THREAD

**Issue by Son-Goku-gpu** [skyhehe123/SA-SSD#66]
Hi, @skyhehe123 Would you like to point out where are the settings of loss weights in your code? I didn't find the \miu and \lambda settings as announced in your paper. Thanks a lot!

**Comment by skyhehe123** [comment]
In my paper, I tune these  hyperparamters because I found the model is quite sensitive to them. However, in my last update where using more robust optimizer and longer training epoch can make the training more stable, the settings of these two parameters therefore are not quite matters.

获取 Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by Son-Goku-gpu** [comment]
Got it. Thanks for your reply!
